packages/opentelemetry-sdk-trace-base/src/export/SimpleSpanProcessor.ts

```diff
@@ -64,43 +63,26 @@ export class SimpleSpanProcessor implements SpanProcessor {
       return;
     }
 
-    const doExport = () =>
-      internal
-        ._export(this._exporter, [span])
-        .then((result: ExportResult) => {
-          if (result.code !== ExportResultCode.SUCCESS) {
-            globalErrorHandler(
-              result.error ??
-                new Error(
-                  `SimpleSpanProcessor: span export failed (status ${result})`
-                )
-            );
-          }
-        })
-        .catch(error => {
-          globalErrorHandler(error);
-        });
+    const pendingExport = this.doExport(span).catch(err =>
+      globalErrorHandler(err)
+    );
+    // Enqueue this export to the pending list so it can be flushed by the user.
+    this._pendingExports.add(pendingExport);
+    pendingExport.finally(() => this._pendingExports.delete(pendingExport));
+  }
 
-    // Avoid scheduling a promise to make the behavior more predictable and easier to test
+  private async doExport(span: ReadableSpan): Promise<void> {
```

Comment:
```suggestion
  private async _doExport(span: ReadableSpan): Promise<void> {
```

The `private` marker isn't a thing in the built `build/src/...` files. It looks like the `_` prefix is being used to add a hint at it being internal/private.